### PR TITLE
[DOC] Fix docstring of `fetch_mixed_gambles` and broken ref in `latest.rst`

### PR DESCRIPTION
**Pull request by NicolasGensollen**
Trying to run a full build locally I had a few errors related mainly to the docstring of ``fetch_mixed_gambles`` (added to the API docs in recently merged #3207).
We didn't catch these since the CI is not running full build anymore.
This PR fixes these issues and improves the rendering of the docstring a little bit.

### CODE REVIEW

File: nilearn/datasets/func.py [hunk 1]
```diff
@@ -1092,32 +1092,36 @@ def fetch_mixed_gambles(n_subjects=1, data_dir=None, url=None, resume=True,
...
-        'zmaps': string list
-            Paths to realigned gain betamaps (one nifti per subject).
-        'gain': ..
-            If make_Xy is true, this is a list of n_subjects * 48
-            Nifti1Image objects, else it is None.
-        'y': array of shape (n_subjects * 48,) or None
-            If make_Xy is true, then this is an array of shape
-            (n_subjects * 48,), else it is None.
+    data : :func:`~sklearn.utils.Bunch`
+        Dictionary-like object, the interest attributes are:
```
Comment (tsalo) [inline nilearn/datasets/func.py:1113]:
```suggestion
        Dictionary-like object, the attributes of interest are:
```

**Review by tsalo — APPROVED** [review]
LGTM. Thanks!

**Review by bthirion — APPROVED** [review]
LGTM, thx !

**Comment by NicolasGensollen** [comment]
Thanks ! Merging....

**Pull request merged**